docs/userguide/canvas.rst

```diff
@@ -959,11 +959,11 @@ Map & Starmap
 -------------
 
 :class:`~celery.map` and :class:`~celery.starmap` are built-in tasks
-that calls the task for every element in a sequence.
+that call the calling task for every element in a sequence.
```

Comment:
Why did you change the phrasing here?

Reply:
"calls" is grammatically incorrect and therefore was changed to "call". 
I added "calling" because I thought it creates a clearer distinction between the "built-in tasks" (i.e. `map` and `starmap`) at the beginning of the sentence, and the task that in executed over the sequence. It's just a proposition.